pkg/controllers/awsloadbalancercontroller/controller.go

```diff
@@ -44,6 +44,10 @@ const (
 	controllerName               = "cluster"
 	controllerSecretName         = "albo-cluster-credentials"
 	controllerServiceAccountName = "albo-cluster-sa"
+	controllerClusterRole        = "albo-cluster-clusterrole"
```

Comment:
Actually the name of the resources should be `aws-load-balancer-controller-$INSTANCE`. You don't need the type in the name because the type is already known.

Reply:
Since all resource names converge on a single name, I'll just create a single const (generic resource name), maybe later on we can add something like [this](https://github.com/openshift/external-dns-operator/blob/main/pkg/operator/controller/names.go)?